### PR TITLE
SCALA-158

### PR DESCRIPTION
**Pull request by mikulskibartosz**

### CODE REVIEW

File: play-scala/async-tasks/app/controllers/AsyncTaskController.scala [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import akka.actor.{ActorRef, ActorSystem}
+import javax.inject._
+import org.joda.time.DateTime
+import play.api.mvc.{Action, AnyContent, BaseController, ControllerComponents}
+import scala.concurrent.ExecutionContext
+import scala.concurrent.duration._
+import scala.language.postfixOps
+
+@Singleton
+class AsyncTaskController @Inject()(val controllerComponents: ControllerComponents, val actorSystem: ActorSystem, @Named("async-job-actor") actor: ActorRef)(implicit ec: ExecutionContext) extends BaseController {
```
Comment (glmartin) [inline play-scala/async-tasks/app/controllers/AsyncTaskController.scala:12]: Let's break this up into multiple lines.

File: play-scala/async-tasks/app/actors/ActorsModule.scala [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package actors
+
+import com.google.inject.AbstractModule
+import play.libs.akka.AkkaGuiceSupport
+
+class ActorsModule extends AbstractModule with AkkaGuiceSupport {
+    override def configure(): Unit = {
+        bindActor(classOf[AsyncTaskInActor], "async-job-actor")
+    }
```
Comment (KevinGilmore) [inline play-scala/async-tasks/app/actors/ActorsModule.scala:9]: Need to use the standard 2-space indent for Scala files.

File: play-scala/async-tasks/conf/logback.xml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    <appender-ref ref="STDOUT" />
+  </appender>
+
+  <logger name="play" level="INFO" />
+  <logger name="application" level="DEBUG" />
+
+  <root level="WARN">
+    <!--<appender-ref ref="ASYNCFILE" />-->
+    <appender-ref ref="ASYNCSTDOUT" />
+  </root>
```
Comment (KevinGilmore) [inline play-scala/async-tasks/conf/logback.xml:40]: Use 4-space indent for XML files.

**Review by KevinGilmore — COMMENTED** [review]
Use 2-space indent in all Scala files and 4-space indent in the XML file

**Review by glmartin — APPROVED** [review]

**Pull request merged**